enigma-js/test/integrationTests/template.02_deploy_voting.js

```diff
@@ -0,0 +1,106 @@
+/* eslint-disable require-jsdoc */
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import forge from 'node-forge';
+import Web3 from 'web3';
+import Enigma from '../../src/Enigma';
+import utils from '../../src/enigma-utils';
+import EnigmaContract from '../../../build/contracts/Enigma';
+import EnigmaTokenContract from '../../../build/contracts/EnigmaToken';
+import VotingETHContract from '../../../build/contracts/VotingETH';
+import * as eeConstants from '../../src/emitterConstants';
+import data from '../data';
+import EthCrypto from 'eth-crypto';
+
+
+forge.options.usePureJavaScript = true;
+
+function sleep(ms) {
+  return new Promise(resolve => setTimeout(resolve, ms));
+}
+
+describe('Enigma tests', () => {
+  let accounts;
+  let web3;
+  let enigma;
+  let votingETHContract;
+  let epochSize;
+  it('initializes', () => {
+    const provider = new Web3.providers.HttpProvider('http://localhost:9545');
+    web3 = new Web3(provider);
+    return web3.eth.getAccounts().then((result) => {
+      accounts = result;
+      enigma = new Enigma(
+        web3,
+        EnigmaContract.networks['4447'].address,
+        EnigmaTokenContract.networks['4447'].address,
+        'http://localhost:3346',
+        {
+          gas: 4712388,
+          gasPrice: 100000000000,
+          from: accounts[0],
+        },
+      );
+      enigma.admin();
+      expect(Enigma.version()).toEqual('0.0.1');
+    });
+  });
+
+  it('initializes VotingETH contract', async () => {
+    votingETHContract = new enigma.web3.eth.Contract(VotingETHContract['abi'],
+      VotingETHContract.networks['4447'].address);
+    expect(votingETHContract.options.address).toBeTruthy();
+  });
+
+  let scTask;
+  let task;
+  const homedir = os.homedir();
+  it('should deploy secret contract', async () => {
+    let scTaskFn = `construct(address)`;
+    let scTaskArgs = [
+      [votingETHContract.options.address, 'address'],
```

Comment:
Question: you are using here ETH 20-byte-long addresses instead of the 32-byte address of the Enigma network like in the Millionaire's contract because wasm is doing nothing with these addresses and is only a pass-through, correct?